### PR TITLE
Added Logistic Regression example

### PR DESCRIPTION
**Pull request by Avinashshah099**
Logistic Regression example in ipynb format. Using Pima Indians Diabetes Database

### CODE REVIEW

File: Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb [hunk 1]
```diff
@@ -0,0 +1,1438 @@
+{
```
Comment (Rishit-dagli) [inline Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb:1]: <p>Could you make sure to follow the guidelines in #46 </p><p>I also see the Markdown <code>#</code>  have not been used properly, you have missed spaces.. I will perform a detailed review on this soon.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (Rishit-dagli) [inline Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb:1]: <p>Could you move</p><blockquote>To understand model performance, dividing the dataset into a training set and a test set is a good strategy. </blockquote><p>to another line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (Rishit-dagli) [inline Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb:1]: <p>Could you move</p><blockquote>First, import the <strong>Logistic Regression</strong> module and create a Logistic Regression classifier object using <strong>LogisticRegression()</strong> function. </blockquote><p>to another line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Comment (Rishit-dagli) [inline Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb:1]: <p>Could you move</p><blockquote>Let's visualize the results of the model in the form of a confusion matrix using matplotlib and seaborn.</blockquote><p>to another line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (Rishit-dagli) [inline Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb:1]: <p>Please remove the blank cell at the end</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Comment (Avinashshah099) [reply Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb:1]: Done

Comment (Avinashshah099) [reply Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb:1]: Done

Comment (Avinashshah099) [reply Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb:1]: Done

Comment (Avinashshah099) [reply Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb:1]: Done

Comment (Avinashshah099) [reply Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb:1]: I have updated it.

File: Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb [hunk 2]
```diff
@@ -0,0 +1,1498 @@
...
```
Comment (Rishit-dagli) [inline Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb:1]: <p>Just a quick post merge fix (since you have made the PR not editable by maintainers), could you use this code to display the notebook buttons, a quick note these links will work well post merge:</p><p> </p><pre class="ql-syntax" spellcheck="false">&lt;table class="tfo-notebook-buttons" align="left"&gt;
  &lt;td&gt;
    &lt;a target="_blank" href="https://colab.research.google.com/github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Logistic%20Regression/Logistic_Regression.ipynb"&gt;&lt;img src="https://www.tensorflow.org/images/colab_logo_32px.png" /&gt;Run in Google Colab&lt;/a&gt;
  &lt;/td&gt;
  &lt;td&gt;
    &lt;a target="_blank" href="https://github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Logistic%20Regression/Logistic_Regression.ipynb"&gt;&lt;img src="https://www.tensorflow.org/images/GitHub-Mark-32px.png" /&gt;View source on GitHub&lt;/a&gt;
  &lt;/td&gt;
&lt;/table&gt;
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (Avinashshah099) [reply Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb:1]: I think the links, that I have used working are working.

Comment (Rishit-dagli) [reply Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb:1]: @Avinashshah099 The links you have used are working but to maintain consistency and ensure uptime (since you have linked to a drive file which could be deleted), I have included links to use Colab for GitHub and enforce those in all notebooks.

Comment (Avinashshah099) [reply Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb:1]: wait, I am updating that.

Comment (Avinashshah099) [reply Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb:1]: done, please have a look

**Review by Rishit-dagli — APPROVED** [review]
This loos good to me, I will go ahead and merge this. Thank you for your contribution 💯 Feel free to support this repo by leaving a :star: !
